src/_base/helm/app/templates/service/elasticsearch/deployment.yaml

```diff
@@ -1,25 +1,26 @@
-{{ if .Values.service.elasticsearch | default .Values.docker.services.elasticsearch.enabled }}
+{{- with .Values.docker.services.elasticsearch -}}
+{{- if $.Values.service.elasticsearch | default .enabled -}}
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ .Values.resourcePrefix }}elasticsearch
+  name: {{ $.Values.resourcePrefix }}elasticsearch
   annotations:
     argocd.argoproj.io/sync-wave: "4"
   labels:
-    app.service: {{ .Values.resourcePrefix }}elasticsearch
+    app.service: {{ $.Values.resourcePrefix }}elasticsearch
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app.service: {{ .Values.resourcePrefix }}elasticsearch
-{{- if .Values.persistence.elasticsearch.enabled }}
+      app.service: {{ $.Values.resourcePrefix }}elasticsearch
+{{- if .enabled }}
```

Comment:
```suggestion
{{- if $.Values.persistence.elasticsearch.enabled }}
```

sorry, I was too aggressively focused to notice every error :)

Reply:
all the errors kubeval would have picked up anyway

Reply:
Yep if the service is enabled (not the case for everything)

Reply:
True :+1: